sources/packages/api/src/route-controllers/assessment/assessment.system-access.controller.ts

```diff
@@ -114,18 +122,15 @@ export class AssessmentSystemAccessController extends BaseController {
    * @returns updates the assessment offering and/or the PIR
    * (Program Info Request) related application data.
    */
-  @ApiResponse({
-    status: HttpStatus.OK,
+  @ApiOkResponse({
```

Comment:
we are no longer using `@ApiOkResponse` right

Reply:
@ApiOkResponse are not forbidden, there was a message in place and I thought that would be good to keep it.
We can add @ApiOkResponse since we add the return type also (in this case is void) and since the comment is something beyond the trivial.

Reply:
But if used, we need to specify the return type here.

Reply:
![image](https://user-images.githubusercontent.com/61259237/164772378-711b8735-c732-4fa8-8f18-2e6d66ff7157.png)